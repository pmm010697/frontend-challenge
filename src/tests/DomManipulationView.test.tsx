import { ThemeProvider } from '@mui/material/styles';
import { lightTheme } from '@src/theme';
import { DomManipulationView } from '@src/views/user/DomManipulationView';
import renderer from 'react-test-renderer';

it('if dom manipulation view snapshot matches', () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={lightTheme}>
        <DomManipulationView />
      </ThemeProvider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
