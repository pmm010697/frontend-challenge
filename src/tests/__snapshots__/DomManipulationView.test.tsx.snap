// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if dom manipulation view snapshot matches 1`] = `
<section>
  <h1
    className="makeStyles-header-1"
  >
    Dummy Structure for DOM Manipulation
  </h1>
  <div>
    <h2
      className="makeStyles-header2-2"
    >
      Not So Important Title
    </h2>
    <p
      className="makeStyles-question-3"
    >
      Asdf
    </p>
    <p
      className="makeStyles-answer-4"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </p>
  </div>
  <div>
    <h2
      className="makeStyles-header2-2"
    >
      More Text
    </h2>
    <p
      className="makeStyles-question-3"
    >
      Budget-to-Beat
    </p>
    <p
      className="makeStyles-btb-6"
    >
       351 â‚¬
    </p>
    <p
      className="makeStyles-answer-4"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </p>
    <p
      className="makeStyles-question-3"
    >
      Are rewards paid out for canceled or changed bookings?
    </p>
    <p
      className="makeStyles-answer-4"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </p>
  </div>
</section>
`;
